specification.md

```diff
@@ -0,0 +1,151 @@
+# Proveth proof-blob format specification
+
+#### Table of Contents
+
+* [Proof kind](#proof-kind)
+* [Proof of transaction inclusion/exclusion](#proof-of-transaction-inclusionexclusion)
+  * [Examples](#examples)
+    * [Transaction 0 of block 5000000 (mainnet)](#transaction-0-of-block-5000000-mainnet)
+    * [Transaction 109 of block 5000000 (mainnet)](#transaction-109-of-block-5000000-mainnet)
+    * [Transaction 6000 of block 6027762 (mainnet)](#transaction-6000-of-block-6027762-mainnet)
+
+----
+
+## Proof kind
+
+Every proveth proof-blob is an [RLP-encoded](https://github.com/ethereum/wiki/wiki/RLP) list.
+The initial element of the list is a uint that specifies the *kind* of proof the proof-blob represents:
```

Comment:
would change the word "initial," to "first"

for some reason that feels less ambiguous, though I couldn't tell you why.

Reply:
I went with "initial" to avoid confusion about one-based vs zero-based indexing.
Switched to "zeroth" and added note explaining that throughout the spec we use zero-based indexing.